lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -291,6 +291,18 @@
     }
 }
 
+.dataDisclaimer-modal{
+  color:#555 !important;
+  padding: 60px !important;
+  line-height: 1.9;
+}
+
+.modal_close{
+  position: absolute;
+  display: block;
+  z-index: 2;
+}
+
```

Comment:
We could name `modal_close` with a more specific name for the data disclaimer modal, but this name it's more generic and could be useful to close any modal.